### PR TITLE
Documentation for the Remootio integration

### PR DESCRIPTION
**Pull request by ivgg-me**
## Proposed change

Documentation for the [Remootio](https://www.remootio.com/) integration introduced with the PR home-assistant/core#63155.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#63155
- Link to parent pull request in the Brands repository: home-assistant/brands#3035
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ivgg-me, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20970).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/remootio.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Remootio
+ha_release: ???
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_domain: remootio
+ha_codeowners:
+  - '@ivgg-me'
+ha_config_flow: true
```
Comment (klaasnicolaas) [inline source/_integrations/remootio.markdown]: missing `description` and `ha_platforms`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/remootio.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This integration lets you control [Remootio](https://www.remootio.com/) controlled garage doors and/or gates through Home Assistant.
+
+### Requirements
+
+To use this integration with your Remootio device it must have at least the software version _2.21_, has to be connected to your Wi-Fi, must have a fixed IP address assigned and 
+a status sensor installed. Furthermore the API access must be enabled on the device.
+
+To enable the API access on your Remootio device you need to access it using the master key via the Remootio app, afterwards go to the _Websocket API_ settings 
+in the _Device software_ section of the device settings and enbale the API with logging.
```
Comment (frenck) [inline source/_integrations/remootio.markdown]:
```suggestion
To use this integration with your Remootio device it must have at least the software version _2.21_, has to be connected to your Wi-Fi, must have a fixed IP address assigned and 
a status sensor installed. Furthermore, the API access must be enabled on the device.

To enable the API access on your Remootio device you need to access it using the master key via the Remootio app, afterward go to the _Websocket API_ settings 
in the _Device software_ section of the device settings and enable the API with logging.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/remootio.markdown (hunk 1 above)
Comment (ivgg-me) [reply source/_integrations/remootio.markdown]: I have created my documentation according the [documentation of Home Assistant](https://developers.home-assistant.io/docs/documenting/create-page). Is this part of the Home Assistant documentation out of date?

Comment (ivgg-me) [reply source/_integrations/remootio.markdown]: Fixed.

Comment (klaasnicolaas) [reply source/_integrations/remootio.markdown]: it's a little bit out of date 😉

File: source/_integrations/remootio.markdown [hunk 3]
```diff
@@ -24,3 +24,29 @@ To enable the API access on your Remootio device you need to access it using the
 in the _Device software_ section of the device settings and enable the API with logging.
 
 {% include integrations/config_flow.md %}
+
+## Events
+
+The integration fires the event `remootio_left_open` if the Remootio controlled garage door or garden gate has been left open for 5 minutes.
```
Comment (frenck) [inline source/_integrations/remootio.markdown]: Can we use device actions instead?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/remootio.markdown (hunk 3 above)
Comment (ivgg-me) [reply source/_integrations/remootio.markdown]: According to the [documentation](https://developers.home-assistant.io/docs/device_automation_action/) device actions allow a user to have a device do something. 

In this case the Remootio device fires an event if the controlled garage door or garden gate has been left open for 5 minutes. Based on this event the integration fires the `remootio_left_open` [Home Assistant event](https://developers.home-assistant.io/docs/integration_events).

Comment (frenck) [reply source/_integrations/remootio.markdown]: > In this case the Remootio device fires an event if the controlled garage door or garden gate has been left open for 5 minutes. 

Device actions are using events under the hood, but provide UI controls to interact with those.

<https://developers.home-assistant.io/docs/device_automation_index>

Comment (ivgg-me) [reply source/_integrations/remootio.markdown]: If you mean that this is really needed for the first version of the integration, please make then an request for change in home-assistant/core#63155. Thank you!

Comment (frenck) [reply source/_integrations/remootio.markdown]: Maybe to much for a first version indeed. however, lets at least not use json here.

Comment (ivgg-me) [reply source/_integrations/remootio.markdown]: > Maybe to much for a first version indeed. however, lets at least not use json here.

Sorry, I don't understand. What you mean with _not use json here_?

Comment (frenck) [reply source/_integrations/remootio.markdown]: Nevermind it is indeed the raw event output.  👍

**Comment by frenck** [comment]
Can you please make sure the build passes?

**Comment by ivgg-me** [comment]
> Can you please make sure the build passes?

I don't understand the errors that occurred during the build. Is there a documentation?

**Comment by ivgg-me** [comment]
> Can you please make sure the build passes?

@frenck: The checks are now running successfully.

**Comment by ivgg-me** [comment]
@frenck What else is necessary for this integration to be merged?

**Comment by frenck** [comment]
@ivgg-me This is the documentation PR, it won't be merged if the parent PR in the core code base isn't merged yet. This PR is thus dependent on and awaiting that.

**Pull request closed without merging**